File: Python Word Examples/ImagesApiDemo/addImage.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.images_api.ImagesApi(configuration)

file_name = 'Image.docx'
image_path = 'input/image.png'
paragraph_path = 'Section/0/Body/0/Paragraph/0'
extra = {
    'folder': 'input',
    'index_in_paragraph': 0,
    'dest_file_path': 'output/AddImage.docx'}
api.add_image(file_name, image_path, paragraph_path, **extra);

File: JavaScript Word Examples/ImagesApiDemo/addImage.js
(function (){
    var appId = "your id";
    var appKey = "your key"; 
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword= require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.ImagesApi(apiClient);

    var name = "Template.docx";
    var imagePath = "input/image.png";
    var paragraphPath = "Section/0/Body/0/Paragraph/0";
    var indexInParagraph = 0;
    var destFilePath = 'output/AddImage.docx';
    var opt = { folder: 'input', indexInParagraph: indexInParagraph };
    instance.addImage(name, imagePath, paragraphPath, destFilePath, opt,
        function (error, data, response) {
            if (error) throw error;
        });
})();